.buildkite/scripts/promote_release_channel.sh

```diff
@@ -2,20 +2,42 @@
 
 set -euo pipefail
 
-channel=$(buildkite-agent meta-data get "release-channel")
```

Comment:
I'm more convinced now that we'll eventually want this as a first class `hab` CLI subcommand, or other options on `hab pkg promote`. In a theoretical sense, promoting a set of packages is likely more safe than doing them individually. Anyway, side tangent, but future work…

Reply:
See https://github.com/habitat-sh/builder/issues/580 :smile: